java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S3752_java.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
I guess this refers to this ticket: SONARJAVA-3376

Reply:
I accepted the change of "type" from "VULNERABILITY" to "SECURITY_HOTSPOT" while we are waiting for SONARJAVA-3376 because I think it makes sense even white the current implementation. But I didn't accept the change of title, it will come with SONARJAVA-3376.